### ISSUE TITLE
bioindustries' liquid-air pulled in when bob's was intended

### ISSUE THREAD

**Issue by wchristian** [kingarthur91/PyCoalTBaA#16]
I'm running a game with all pymods, and bioindustries, but no bobs or angels.

The overrides.lua in the bobs directy replaces steam in a bunch of recipes with liquid-air, using numbers intended for bobs. However this replace ALSO fires if bob isn't loaded, but bioindustries is, because bio has a very very slow process for making liquid-air.

Not sure how to fix this, wouldn't it be most correct to have the entire overrides.lua file there wrapped in `if mods["bobores"] then` to be safe?

**Comment by kingarthur91** [comment]
Not necessarily as it could be depending on more than bob ores. To get it to only work when bob is enable would want to use `if mods["boblibrary"] then`. 

Or more accurately to wrap the part replacing the steam with liquid-air in `if not mods["bioindustry"] or mods["boblibarary"] then`. So that it will fire when bioindustry isnt active or if bob mods is active.

**Comment by kingarthur91** [comment]
also i found 2 recipes that replace steam with liquid air. wouldnt call that a bunch. so let me know if there are more that im not seeing somehow but ive added the check so it shouldnt do that anymore

**Comment by wchristian** [comment]
First off, thanks for the quick fix. I applied it, and found it didn't have the effect hoped for, on account of boblibrary being loaded for the functions. I was able to disable it without breaking anything and that did the thing. So not sure if you wanna make that check a little more specific.

As for the bunch: I spoke a bit rash. In "what is it really used for" i saw 7 recipes affected. Three of them were from bioindustries itself. The other four were:

  - anthraquinone
  - tpa
  - glycerol-hydrogen
  - tar-gasification

You addressed the first two directly, but somehow another fix you did also fixed the latter two, so yay.
